packages/grid/src/GridMetricCalculator.ts

```diff
@@ -1763,6 +1764,10 @@ export class GridMetricCalculator {
       cellHorizontalPadding * 2
     );
 
+    if (isDataBarGridModel(model)) {
+      return columnWidth + 90;
+    }
```

Comment:
I put it in the `iterateAllItems` but it will add 90 each time a dataBar cell is found. Could it be added afterwards? Instead of `isDataBarGridModel(model)`, it would be `if (renderType === 'dataBar')`.

Reply:
```
row => {
        const modelRow = this.getModelRow(row, state);
        const text = model.textForCell(modelColumn, modelRow);
        const cellRenderType = model.renderTypeForCell(modelColumn, modelRow);

        if (text) {
          const cellPadding = cellHorizontalPadding * 2;
          columnWidth = Math.max(
            columnWidth,
            text.length * fontWidth + cellPadding
          );
        }

        if (cellRenderType === 'dataBar') {
          columnWidth += 90;
        }
      }
```

Do you mean like this?